files/es/web/api/notification/dir/index.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Notification.dir
+slug: Web/API/notification/dir
+tags:
+  - API
+  - Notification
+  - Notifications
+  - Notifications API
+  - Property
+  - Reference
+  - dir
+translation_of: Web/API/Notification/dir
+browser-compat: api.Notification.dir
+---
+{{APIRef("Web Notifications")}}{{AvailableInWorkers}}{{SeeCompatTable()}}
+
+La propiedad `dir` de solo lectura de la interfaz {{domxref("Notification")}} indica la direccion del texto en la notificación, como se especifica en la opción `dir` del constructor {{domxref("Notification.Notification","Notification()")}}.
+
+## Sintaxis
+
+```js
+var direction = Notification.dir;
+```
+
+### Valor
+
+Un {{domxref("DOMString")}} especifica la dirección del texto, los posibles valores pueden ser:
+
+- `auto`: Aplica el comportamiento del idioma del navegador (por defecto).
+- `ltr`: El texto es desplegado de izquierda a derecha.
+- `rtl`: El texto es desplegado de derecha a izquierda.
+
+> **Nota:** La mayoría de los navegadores parecen ignorar las intrucciones explicitas ltr y rtl, y solo van a las proveídas por el navegador.
+
+## Ejemplos
+
+El siguiente código ejecuta una notificación; se crea un simple objeto `options`, luego la notificación se dispara usando el constructor `Notification()`.
+
+```js
+var options = {
+  body: 'Do you like my body?',
+  dir: 'rtl'
+}
+
+var n = new Notification('Notificación de prueba',options);
+
+console.log(n.dir) // Deberia retornar 'rtl'
```

Comment:
Also not in en-US for some reason. I find it strange and inconsistent since a semi-colon was used for the rest of the example code, might as well fix it here (up to you though)?

```suggestion
console.log(n.dir); // Deberia retornar 'rtl'
```

Reply:
Agree ;)

Reply:
Regarding this, ES6 code validators such esLint or Eshint had as rule avoid the use of semicolons, just in certain scenario it is necessary to include them. Not to mention that each framework out there has their own rules, as well as commonJS and modernJS does. How are we dealing with this?

Reply:
Here, we follow [MDN's style/code guidelines](https://developer.mozilla.org/en-US/docs/MDN/Guidelines/Code_guidelines/JavaScript). Quoting their JavaScript code guidelines:

> All statements must end with semicolons (";"). We require them in all of our code samples even though they're technically optional in JavaScript because we feel that it leads to code that is clearer and more precise about where each statement ends.